lib/ace/snippets/papeeria_latex.snippets

```diff
@@ -0,0 +1,45 @@
+#DOCUMENT
+# \begin{}...\end{}
+snippet \begin{}...
+	\begin{${1:env}}
+		${2}
+	\end{$1}
+snippet \begin{itemize}...
+	\begin{itemize}
+		\item ${1}
+	\end{itemize}
+snippet \begin{equation}...
+	\begin{equation}
+		${1}
+	\end{equation}
+# Equation
+snippet \begin{equation*}...
+	\begin{equation*}
+		${1}
+	\end{equation*}
+snippet \begin{enumerate}...
+	\begin{enumerate}
+		\item ${1}
+	\end{enumerate}
+# Itemize
+snippet \begin{itemize}...
+	\begin{itemize}
+		\item ${1}
+	\end{itemize}
+# Description
+snippet \begin{description}...
+	\begin{description}
+		\item[${1}] ${2}
+	\end{description}
+snippet \begin{split}...
+	\begin{split}
+		${1}
+	\end{split}
+
+#math
```

Comment:
maybe remove math snippets for a while? It would be great to support them, however they should be used in math mode only, and we do not yet detect if we're in math mode or not.